### ISSUE TITLE
Error on install

### ISSUE THREAD

**Issue by coqmos** [rpkamp/mailhog-mink-behat-context#13]
On install of the package i get the following

  Problem 1
    - rpkamp/mailhog-client 0.3.5 requires php-http/client-implementation ^1.0 -> no matching package found.
    - rpkamp/mailhog-client 0.3.4 requires php-http/client-implementation ^1.0 -> no matching package found.
    - rpkamp/mailhog-client 0.3.3 requires php-http/client-implementation ^1.0 -> no matching package found.
    - rpkamp/mailhog-client 0.3.2 requires php-http/client-implementation ^1.0 -> no matching package found.
    - rpkamp/mailhog-behat-extension 0.3.5 requires rpkamp/mailhog-client ^0.3.2 -> satisfiable by rpkamp/mailhog-client[0.3.2, 0.3.3, 0.3.4, 0.3.5].
    - Installation request for rpkamp/mailhog-behat-extension ^0.3.5 -> satisfiable by rpkamp/mailhog-behat-extension[0.3.5].

Any tips on how to solve?

**Comment by rpkamp** [comment]
You need to also add an HTTP client while requiring this package, example:

```
composer require php-http/message guzzlehttp/psr7 rpkamp/mailhog-mink-behat-context
```

You need to require them either all at once, or require `rpkamp/mailhog-mink-behat-context` after you've required the other ones.

See http://docs.php-http.org/en/latest/httplug/users.html


I will update the README to add this.

Thanks for your report!

**Comment by brooke-heaton** [comment]
I tried the solution above but it did not resolve that issue. Composer require runs fine, but when running tests, I get:

```
[Http\Discovery\NotFoundException]                                                                                                         
  No HTTPlug clients found. Make sure to install a package providing "php-http/client-implementation". Example: "php-http/guzzle6-adapter".  
```

**Comment by brooke-heaton** [comment]
And of course trying to do a `composer require php-http/guzzle6-adapter` simply results in this mess:

```
  Problem 1
    - Conclusion: don't install php-http/guzzle6-adapter v2.0.1
    - Conclusion: don't install php-http/guzzle6-adapter v2.0.0
    - don't install php-http/guzzle6-adapter 2.x-dev|install php-http/guzzle6-adapter dev-master
    - Conclusion: remove php-http/httplug v1.1.0
    - Installation request for php-http/guzzle6-adapter ^2.0 -> satisfiable by php-http/guzzle6-adapter[2.x-dev, v2.0.0, v2.0.1].
    - Conclusion: don't install php-http/httplug v1.1.0
    - php-http/guzzle6-adapter dev-master requires php-http/httplug ^2.0 -> satisfiable by php-http/httplug[2.0.x-dev, v2.0.0].
    - Can only install one of: php-http/httplug[2.0.x-dev, v1.1.0].
    - Can only install one of: php-http/httplug[v2.0.0, v1.1.0].
    - Installation request for php-http/httplug (locked at v1.1.0) -> satisfiable by php-http/httplug[v1.1.0].
```

**Comment by brooke-heaton** [comment]
You need to update to php-http/httplug 2.0 for this to work. :(

**Comment by rpkamp** [comment]
This was fixed in rpkamp/mailhog-client version 0.3.6

https://github.com/rpkamp/mailhog-client/releases/tag/0.3.6
